crates/p2p/src/swarm/behavior.rs

```diff
@@ -1,4 +1,7 @@
 //! Request-Response [`Behaviour`] and [`NetworkBehaviour`] for the P2P protocol.
+// Allow missing docs to avoid 'missing documentation for a variant' error from NetworkBehaviour
+// derive.
+#![allow(missing_docs)]
```

Comment:
Can you add that `allow` to the specific offending line `#` and not trigger moudule-level with `#!`?